### PR TITLE
FIX: Send sms&email alerts when BTCPay server is down

### PR DESCRIPTION
**Pull request by ilhanmercan**

### CODE REVIEW

File: src/billing/management/commands/btcpay_verify.py [hunk 1]
```diff
@@ -31,6 +31,11 @@ def handle(self, *args, **options):
         )
 
         while True:
+            try:
+                client.get_rate("USD")
+            except:
+                self._send_sms_and_email_alert()
```
Comment (vesellov) [inline src/billing/management/commands/btcpay_verify.py]: i think if `get_rates()` failed we should not continue reading invoices and do `time.sleep(60)` ?
also please always add `logger.exception()` to all try - except statements!

File: src/main/params_example.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
 # EMAIL_PORT = 587
 # EMAIL_USE_TLS = True
 # EMAIL_USE_SSL = False
+# EMAIL_ADMIN = 'admin@zenaida.cate.ai'
```
Comment (vesellov) [inline src/main/params_example.py]: please do not hard-code domain name... we should use `settings.SITE_BASE_URL` probably

Comment (ilhanmercan) [reply src/main/params_example.py]: changed it

File: src/billing/management/commands/btcpay_verify.py (hunk 1 above)
Comment (ilhanmercan) [reply src/billing/management/commands/btcpay_verify.py]: changed it

**Pull request merged**